docs/usage/configuration/config_documentation.md

```diff
@@ -179,7 +179,7 @@ This will tell other servers to send traffic to port 443 instead.
 
 This option currently defaults to false.
 
-See https://matrix-org.github.io/synapse/latest/delegate.html for more
+   See [Delegation of incoming federation traffic](https://matrix-org.github.io/synapse/latest/delegate.html) for more
```

Comment:
As Dirk mentioned, you'll want to use relative links to local files in the documentation directory. This means writing links in the form:

```
[my awesome link](../../awesome.md)
```

`mdbook` (the tool we use to generate the documentation HTML) will then automatically create the `*.html` link for you :)